src/Cameras/VR/babylon.webVRCamera.ts

```diff
@@ -125,6 +128,22 @@ module BABYLON {
                 if (this._attached) {
                     this.getEngine().enableVR();
                 }
+                if(this._useStandingMatrix){
+                    if (!this._vrDevice.stageParameters.sittingToStandingTransform) {
+                        this._useStandingMatrixCallback(false);
+                    }else{
+                        this._standingMatrix = new Matrix();

```

Comment:
I believe this function should be executed anyhow, storing the standingMatrix.
if this._useStandingMatrix is set to true, it should be used. if not, then no.

Reply:
(I mean the entire scope after "if(this._useStandingMatrix){" should be executed